angular-breadcrumb/angular-breadcrumb.d.ts

```diff
@@ -14,7 +14,7 @@ declare namespace angular.ui {
             /**
              * Override the parent state (only for the breadcrumb)
              **/
-            parent?: string;
+            parent?: string|Function;
```

Comment:
This should be written as an actual function signature. Only use `Function` as a last resort.